src/pykeen/triples/triples_factory.py

```diff
@@ -1346,3 +1348,78 @@ def splits_similarity(a: Sequence[CoreTriplesFactory], b: Sequence[CoreTriplesFa
     steps = splits_steps(a, b)
     n = sum(tf.num_triples for tf in a)
     return 1 - steps / n
+
+
+def get_mapped_triples(
```

Comment:
if you're confident this works right and doesn't need more unit tests then :shipit:

Reply:
testing does not hurt :slightly_smiling_face: 
https://github.com/pykeen/pykeen/pull/1089/commits/c6ed508f2622362dbf885338707be40cf7bb3cb3